core/services/ocrcommon/config.go

```diff
@@ -25,14 +24,17 @@ func ParseBootstrapPeers(peers []string) (bootstrapPeers []commontypes.Bootstrap
 }
 
 // GetValidatedBootstrapPeers will error unless at least one valid bootstrap peer is found
-func GetValidatedBootstrapPeers(specPeers []string, configPeers []commontypes.BootstrapperLocator) ([]commontypes.BootstrapperLocator, error) {
+func GetValidatedBootstrapPeers(lggr logger.Logger, specPeers []string, configPeers []commontypes.BootstrapperLocator) ([]commontypes.BootstrapperLocator, error) {
 	bootstrapPeers, err := ParseBootstrapPeers(specPeers)
 	if err != nil {
 		return nil, err
 	}
 	if len(bootstrapPeers) == 0 {
 		if len(configPeers) == 0 {
-			return nil, errors.New("no bootstrappers found")
+			lggr.Warn("no bootstrappers found")
```

Comment:
Passing a logger through a func like this is typically a code smell. Can the caller log this instead?

Reply:
removed passing in log